src/cljs/rems/actions/add_licenses.cljs

```diff
@@ -0,0 +1,116 @@
+(ns rems.actions.add-licenses
+  (:require [clojure.string :as str]
+            [re-frame.core :as rf]
+            [rems.actions.action :refer [action-button action-form-view action-comment button-wrapper collapse-action-form]]
+            [rems.autocomplete :as autocomplete]
+            [rems.status-modal :as status-modal]
+            [rems.text :refer [text]]
+            [rems.util :refer [fetch post!]]))
+
+(rf/reg-fx
+ ::fetch-licenses
+ (fn [[user on-success]]
+   (fetch "/api/licenses"
+          {:handler on-success
+           :headers {"x-rems-user-id" (:eppn user)}})))
```

Comment:
we should have a `fetch-as-current-user` util that injects this header. perhaps in another PR

Reply:
Good idea, I'll add a task for it

Reply:
Actually, I'll just remove it in another PR since it's not needed (comes from the session)